memgpt/schemas/message.py

```diff
@@ -62,6 +62,8 @@ class Message(BaseMessage):
     id: str = BaseMessage.generate_id_field()
     role: MessageRole = Field(..., description="The role of the participant.")
     text: str = Field(..., description="The text of the message.")
+    # Field mm_content is only used when role is 'user'. It needs to be mapped to MultiModalMessage
```

Comment:
Same comment here - deprecating `text` and replacing with a new `content` that matches OpenAI